src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,182 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useState } from "react"
 
 export const PreferencesApp: React.FC = () => {
+  const [uncheckedAll, setUncheckedAll] = useState(false)
+
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <Column
+        span={12}
+        pb={2}
+        pr={12}
+        mt={2}
+        display="flex"
+        justifyContent="space-between"
+      >
+        <Flex p={2} flexDirection="column">
+          <Box flexBasis={"50%"}>
```

Comment:
In general: use `GridColumns` + `Column` for laying out things. You're going to very, very rarely be setting a width manually in Force.

Reply:
#superminor - strings don't need `{ }` around them

Reply:
@damon So would the GridColumns + Column would replace the usage of `flexBasis="50%"`?

Reply:
Yeah. Take a look at all the other settings pages to see. We've been bounding them to not quite 50%.

```tsx
<GridColumns>
  <Column span={8}>
    // ...
  </Column>
</GridColumns>
```

Reply:
Updated this.